site/gdocs/components/People.scss

```diff
@@ -0,0 +1,6 @@
+.people {
```

Comment:
Good question, I should have mentioned it in the PR description. I did consider it, but decided against it because we'll have three different layouts for people configurable in archie: 1, 2 and 4 columns and on small screens they need to collapse into a single column row by row, e.g. first all the people from the first row, then all people from the second row etc.

Reply:
Okay! Can we take a look at it together when you get around to implementing the stacking layout? I just wanna make sure I understand if there's a way in which the grid system isn't working for us 👍

Reply:
On a second thought, we might be able to use the grid system at least partially. You'll either see in the next PR or I'll talk to you before, if it's gonna be a significant amount of work.